src/forms/TextInput.jsx

```diff
@@ -5,79 +5,62 @@ import cx from 'classnames';
 /**
  * @module TextInput
  */
-class TextInput extends React.Component {
+function TextInput(props) {
```

Comment:
can i be obnoxious and ask this to be written es6'y?
`const TextInput = props => { ... };`